Formula/tor.rb

```diff
@@ -39,15 +39,6 @@ def install
     system "make", "install"
   end
 
-  def caveats; <<-EOS.undent
```

Comment:
> Please either quit with these caveats or quit with the comments that precede them, thanks. If you believe your opinion to be genuinely ignored: self-censor. If you think you'll be listened to: don't self-sabotage your own argument.

For flip sake Mike, I wasn't taking a dig at you, and I wish you'd stop reacting like I'm out to get you every time I breathe something you consider negative.

My comment was purely a notation that whilst I thought my comment would be taken on board I didn't expect it to shift the winds significantly enough for the work here to be discarded. I don't expect my dislike of President Trump's anti-immigration rhetoric to make a single drop of difference, and I'd have to be an idiot to think otherwise, but it doesn't mean I'm going to sit there and go _"Yes I love this, it's great"_.